contracts/GNTPaymentChannels.sol

```diff
@@ -68,13 +68,13 @@ contract GNTPaymentChannels is ReceivingContract {
         pure
         returns (bool)
     {
-        return _owner == ecrecover(keccak256("\x19Ethereum Signed Message:\n72", _owner, _receiver, _value), _v, _r, _s);
+        return _owner == ecrecover(keccak256(abi.encodePacked("\x19Ethereum Signed Message:\n72", _owner, _receiver, _value)), _v, _r, _s);
```

Comment:
Is this exactly the same as was before?

Reply:
Yup, even compiler suggests using exactly that to achieve v0.4 compatibility. Also tests still pass.